src/validation/tropopause/get_dtdz.cpp

```diff
@@ -0,0 +1,35 @@
+// mam4xx: Copyright (c) 2022,
+// Battelle Memorial Institute and
+// National Technology & Engineering Solutions of Sandia, LLC (NTESS)
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <mam4xx/mam4.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <skywalker.hpp>
+#include <validation.hpp>
+
+using namespace skywalker;
+using namespace mam4;
+
+void get_dtdz(Ensemble *ensemble) {
+  ensemble->process([=](const Input &input, Output &output) {
+    const Real pm = input.get_array("pm")[0];
+    const Real pmk = input.get_array("pmk")[0];
+    const Real pmid1d_up = input.get_array("pmid1d_up")[0];
+    const Real pmid1d_down = input.get_array("pmid1d_down")[0];
+    const Real temp1d_up = input.get_array("temp1d_up")[0];
+    const Real temp1d_down = input.get_array("temp1d_down")[0];
+    // const Real cnst_kap = input.get_array("cnst_kap")[0];
```

Comment:
Can we delete these two lines, or do you think we might need to re-add them later?

Reply:
`cnst_kap`, `cnst_faktor`, and `cnst_ka1` are currently constants at the top of `tropopause.hpp`, but are also provided in the input file. I have opted to calculate them as constants in the hpp file but could change that

Reply:
I think your intuition is good here. Let's keep constants, uhh, constant, and not expose them as variable parameters. :-) Okay to delete these lines in my opinion.